src/components/WalletModal/WalletModal.tsx

```diff
@@ -75,157 +78,208 @@ const WalletModal: React.FC<ModalProps> = ({ isOpen, onDismiss }) => {
+              link={`https://etherscan.io/address/${tokenAddresses.dataTokenAddress}`}
+              value={displayFromWei(dataBalance)}
+            />
+          </Box>
+        </Split>
+        <Spacer />
+        <Separator />
+        <Spacer />
+        <Split>
+          <Box row>
+            <FancyValue
+              icon={{
+                alt: 'ETH2x-FLI Icon',
+                src: 'https://set-core.s3.amazonaws.com/img/portfolios/eth2x_fli.svg',
+              }}
+              label='ETH 2x FLI Balance'
+              link={`https://etherscan.io/address/${tokenAddresses.eth2xfliTokenAddress}`}
+              value={displayFromWei(ethfliBalance)}
+            />
+          </Box>
+          <Box row>
+            <FancyValue
+              icon={{
+                alt: 'BTC2x-FLI Icon',
+                src: 'https://set-core.s3.amazonaws.com/img/portfolios/fli_btc.svg',
+              }}
+              label='BTC 2x FLI Balance'
+              link={`https://etherscan.io/address/${tokenAddresses.btc2xfliTokenAddress}`}
+              value={displayFromWei(btcfliBalance)}
+            />
+          </Box>
+        </Split>
+        <Spacer />
+        <Separator />
+        <Spacer />
+        <Split>
+          <Box row>
+            <FancyValue
+              iconComponent={ethDpiTokenIcon}
+              label='Uniswap ETH/DPI LP balance'
+              link={`https://etherscan.io/address/${tokenAddresses.uniswapEthDpiLpTokenAddress}`}
+              value={displayFromWei(uniswapEthDpiLpBalance)}
+            />
+          </Box>
+          <Box row>
+            <FancyValue
+              icon={{
+                alt: 'Staked Uniswap ETH/DPI LP Icon',
+                src: 'https://set-core.s3.amazonaws.com/img/coin-icons/uni_lp.svg',
+              }}
+              label='Staked Uniswap ETH/DPI LP'
+              link={`https://etherscan.io/address/${tokenAddresses.farmTwoAddress}`}
+              value={displayFromWei(totalStakedEthDpiLpBalance)}
+            />
+          </Box>
+        </Split>
+        <Spacer />
+        <Split>
+          <Box row>
+            <FancyValue
+              iconComponent={ethMviTokenIcon}
+              label='Uniswap ETH/MVI LP balance'
+              link={`https://etherscan.io/address/${tokenAddresses.uniswapEthMviLpTokenAddress}`}
+              value={displayFromWei(uniswapEthMviLpBalance)}
+            />
+          </Box>
+          <Box row>
+            <FancyValue
+              icon={{
+                alt: 'Staked Uniswap ETH/MVI LP Icon',
+                src: 'https://set-core.s3.amazonaws.com/img/coin-icons/uni_lp.svg',
+              }}
+              label='Staked Uniswap ETH/MVI LP'
+              link={`https://etherscan.io/address/${tokenAddresses.mviStakingRewardsAddress}`}
+              value={displayFromWei(stakedUniswapEthMviLpBalance)}
+            />
+          </Box>
+        </Split>
+      </>
+    )
+  }
+
+  const polygonModalContent = () => {
```

Comment:
Can we also label the DPI / MVI addresses as Polygon as well. Maybe `DPI Polygon Balance`